{
  "incident": {
    "components": [
      {
        "created_at": "2017-01-31T20:05:05.370Z",
        "description": "Performance of git clones, pulls, pushes, and associated operations",
        "group_id": null,
        "id": "8l4ygp009s5s",
        "name": "Git Operations",
        "page_id": "kctbh9vrtdwd",
        "position": 1,
        "showcase": false,
        "start_date": null,
        "status": "partial_outage",
        "updated_at": "2025-09-16T18:02:37.848Z"
      }
    ],
    "created_at": "2025-09-16T17:55:11.551Z",
    "id": "nh22ywy9dh84",
    "impact": "minor",
    "impact_override": "minor",
    "incident_updates": [
      {
        "affected_components": [
          {
            "code": "8l4ygp009s5s",
            "name": "Git Operations",
            "new_status": "partial_outage",
            "old_status": "operational"
          }
        ],
        "body": "Git Operations is experiencing degraded performance. We are continuing to investigate.",
        "created_at": "2025-09-16T18:02:37.897Z",
        "custom_tweet": null,
        "deliver_notifications": true,
        "display_at": "2025-09-16T18:02:37.897Z",
        "id": "k0kjgslkh5vf",
        "incident_id": "nh22ywy9dh84",
        "status": "investigating",
        "tweet_id": null,
        "twitter_updated_at": null,
        "updated_at": "2025-09-16T18:02:37.897Z",
        "wants_twitter_update": false
      },
      {
        "affected_components": null,
        "body": "A recent change to our API routing inadvertently added an authentication requirement to the anonymous route for LFS requests. We're in the process of fixing the change, but in the interim retrying should eventually succeed.",
        "created_at": "2025-09-16T17:55:32.201Z",
        "custom_tweet": null,
        "deliver_notifications": true,
        "display_at": "2025-09-16T17:55:32.201Z",
        "id": "vf1g0nx7mvqf",
        "incident_id": "nh22ywy9dh84",
        "status": "investigating",
        "tweet_id": null,
        "twitter_updated_at": null,
        "updated_at": "2025-09-16T17:55:32.201Z",
        "wants_twitter_update": false
      },
      {
        "affected_components": null,
        "body": "We are currently investigating this issue.",
        "created_at": "2025-09-16T17:55:11.587Z",
        "custom_tweet": null,
        "deliver_notifications": true,
        "display_at": "2025-09-16T17:55:11.587Z",
        "id": "5xtdlsvqyr8v",
        "incident_id": "nh22ywy9dh84",
        "status": "investigating",
        "tweet_id": null,
        "twitter_updated_at": null,
        "updated_at": "2025-09-16T17:55:11.587Z",
        "wants_twitter_update": false
      }
    ],
    "metadata": {},
    "monitoring_at": null,
    "name": "Unauthenticated LFS requests for public repos are returning unexpected 401 errors",
    "page_id": "kctbh9vrtdwd",
    "postmortem_body": null,
    "postmortem_body_last_updated_at": null,
    "postmortem_ignored": false,
    "postmortem_notified_subscribers": false,
    "postmortem_notified_twitter": false,
    "postmortem_published_at": null,
    "reminder_intervals": null,
    "resolved_at": null,
    "scheduled_auto_completed": false,
    "scheduled_auto_in_progress": false,
    "scheduled_for": null,
    "scheduled_remind_prior": false,
    "scheduled_reminded_at": null,
    "scheduled_until": null,
    "shortlink": "https://stspg.io/gpf81479c8n8",
    "started_at": "2025-09-16T17:55:11.545Z",
    "status": "investigating",
    "updated_at": "2025-09-16T18:02:37.901Z"
  },
  "meta": {
    "generated_at": "2025-09-16T18:07:18.013Z"
  },
  "page": {
    "id": "kctbh9vrtdwd",
    "status_description": "Minor Service Outage",
    "status_indicator": "minor"
  }
}
